test/conformance/tests/test_sriov_operator.go

```diff
@@ -200,10 +200,13 @@ var _ = Describe("[sriov] operator", func() {
 				discoveryFailed = node == "" || resourceName == "" || numVfs < 5
 			} else {
 				node = sriovInfos.Nodes[0]
-				createVanillaNetworkPolicy(node, sriovInfos, numVfs, resourceName)
+				sriovDevice = createVanillaNetworkPolicy(node, sriovInfos, numVfs, resourceName)
```

Comment:
Value of `sriovDevice`  is never read. What about reverting the BeforeEach part?

Reply:
correct, this is a leftover of a previous implementation. Don't understand what part you want me to revert. What I'm adding is something missing. After creating the general sriov policy, the info about interfaces is not updated.